File: sample-app/src/main/java/io/github/lazymockbean/sampleapp/service/SampleServiceC.java
package io.github.lazymockbean.sampleapp.service;

import org.springframework.stereotype.Service;

@Service
public class SampleServiceC {
    public String getSample1() {
        return "sample-C-1";
    }

    public String getSample2() {
        return "sample-C-2";
    }
}

File: sample-app/src/main/kotlin/io/github/taesk/lazymockbean/sampleapp/service/SampleServiceC.kt
package io.github.taesk.lazymockbean.sampleapp.service

import org.springframework.stereotype.Service

@Service
class SampleServiceC {
    fun getSample1(): String {
        return "sample-C-1"
    }

    fun getSample2(): String {
        return "sample-C-2"
    }
}
